tests/integration/courses.test.js

```diff
@@ -867,16 +867,23 @@ describe('COURSES ROUTES - PUT /courses/{_id}', () => {
       expect(response.statusCode).toBe(400);
     });
 
-    it('should return 403 if course is archived', async () => {
-      const payload = { misc: 'new name' };
-      const response = await app.inject({
-        method: 'PUT',
-        url: `/courses/${archivedCourse}`,
-        headers: { Cookie: `alenvi_token=${authToken}` },
-        payload,
-      });
+    const payloads = [
+      { misc: 'new name' },
+      { trainer: new ObjectID() },
+      { contact: { name: 'new name', phone: '0101010101', email: 'test@email.com' } },
```

Comment:
Qu'est-ce qui fait que tu testes ces trois champs en une fois, et les autres un a un ?

Reply:
parce que ça me paraissait pas bien gênant de tester les trois en même temps et que sinon ça fait 3 tests avec le même nom (should return 403 if course is archived (update contact)) mais du coup j'ai corrigé

Reply:
Pour eviter d'avoir trois fois le meme nom tu peux creer le payload ainsi : `{ 'contact.name': 'new name' }` il me semble. 
Je suis d'accord que ce n'est pas forcement genant mais je me disais que pour etre coherent avec le fait qu'on test tous les autres champs un a un on pouvait les tester aussi ainsi (et aussi parce qu'il n'y en a pas trop d'imbriqué ^^).